htdocs/compta/bank/account.php

```diff
@@ -397,7 +397,7 @@
             }
         }
 
-		if ($object->type != 2 && $object->rappro) 
+		if ($object->canBeConciliated())
```

Comment:
The function does not return a boolean but an int to say why it can'b be conciliated. So you should test on 
$object->canBeConciliated() > 0